src/main/java/com/minecolonies/rankup/modules/databases/DatabaseModule.java

```diff
@@ -0,0 +1,26 @@
+package com.minecolonies.rankup.modules.databases;
+
+import com.minecolonies.rankup.modules.databases.config.DatabaseConfigAdapter;
+import com.minecolonies.rankup.qsml.modulespec.ConfigurableModule;
+import org.spongepowered.api.service.sql.SqlService;
+import uk.co.drnaylor.quickstart.annotations.ModuleData;
+
+@ModuleData(id = DatabaseModule.ID, name = "Databases")
+public class DatabaseModule extends ConfigurableModule<DatabaseConfigAdapter>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1258)